### ISSUE TITLE
Audio API ignores portals

### ISSUE THREAD

**Issue by mrgharabaghi** [altmp/altv-issues#1898]
### Description of the problem

Audio API doesn't work as it should.

### Reproduction steps

```js
import * as alt from 'alt-client';

let myAudio;
let myVehicleId;

alt.on('keyup', async (key) => {
    // Q == 81
    if (key === 81) {
        if (myAudio instanceof alt.Audio) {
            myAudio.reset();
        }

        myVehicleId = alt.Player.local.vehicle;
        myAudio = await new alt.Audio('@audio-nikto-assets/sound/fox.mp3', 1);

        setTimeout(async () => {
            let audio = await new alt.AudioOutputAttached(myVehicleId, alt.hash('music'));
            myAudio.addOutput(audio);
            myAudio.looped = false;
            myAudio.play();
        }, 1000);
    }
});
```

### Expected behaviour

It should work as before.

https://github.com/altmp/altv-issues/assets/40915667/ddc23f40-68b8-4e5a-b7b0-e94f40e896a5

### Operating system

Windows 10

### Version

dev/15.0-dev344

### Reproduction tested

- [X] I confirm that I have made sure that this issue is also present on the newest dev version

**Comment by martonp96** [comment]
@mrgharabaghi can you record a video?

**Comment by mrgharabaghi** [comment]
Sure

**Comment by mrgharabaghi** [comment]
Fixed in `dev/15.0-dev346`
Thank you @martonp96
I thought that still doesn't work.
